addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -1965,7 +2086,7 @@ X[]
                 await testEditor(BasicEditor, {
                     contentBefore: '<p>ab<b class="oe_unremovable">[cd]</b>ef</p>',
                     stepFunction: deleteBackward,
-                    contentAfter: '<p>ab<b class="oe_unremovable">[\u200B]</b>ef</p>',
+                    contentAfter: '<p>ab<b class="oe_unremovable">[]\u200B</b>ef</p>',
```

Comment:
This is so much better !

Reply:
well ... 
https://github.com/odoo/odoo/pull/84187#discussion_r803649249